components/brave_shields/browser/https_everywhere_service.cc

```diff
@@ -146,7 +146,8 @@ bool HTTPSEverywhereService::GetHTTPSURL(
     const GURL* url, const uint64_t& request_identifier,
     std::string& new_url) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
-  base::AssertBlockingAllowed();
+  base::ScopedBlockingCall scoped_blocking_call(
```

Comment:
same thing here with the thread. Not a problem with this PR, but I didn't have to do this for rewards leveldb stuff

Reply:
because this is a change in the 70 upgrade.  I'll have to do this when I rebase on top of rewards.